src/assets/style/landmark/_stripView.less

```diff
@@ -18,18 +18,19 @@
     right: -390px;
 }
 
+.stripTitle {
```

Comment:
I'd like to see a little extra padding around the title.  Try something like this:
```css
.stripTitle {
    background-color: #222222;
    color: #ffffff;
    font-size: 1.25em;
    padding: 5px 0;
    text-align: center;
}
```

I'd also like to see existing color vars used here instead of inline hex codes.

Reply:
Fixed in latest commit